specs/experimental/custom-gas-token.md

```diff
@@ -82,6 +82,7 @@ The gas paying token MUST NOT:
 - Have out of band methods for modifying balance or allowance
 - Have a `name()` that is more than 32 bytes long
 - Have a `symbol()` this is more than 32 bytes long
+- Be a [double entrypoint token](https://stermi.xyz/blog/ethernaut-challenge-24-solution-double-entry-point)
```

Comment:
Potentially more concise definition, don't feel strongly though so not a blocker: https://chainsecurity.com/denial-of-service-attacks-in-defi-the-balancer-synthetix-case/#dep
```suggestion
- Be a [double entrypoint token](https://chainsecurity.com/denial-of-service-attacks-in-defi-the-balancer-synthetix-case/#dep)
```